python/pyspark/streaming/context.py

```diff
@@ -260,6 +260,12 @@ def textFileStream(self, directory):
         """
         return DStream(self._jssc.textFileStream(directory), self, UTF8Deserializer())
```

Comment:
Should this just call the other method with arg 1 instead of duplicating the logic?